src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -230,6 +230,12 @@ MessagePtr TwitchMessageBuilder::build()
                 // Actually just text
                 QString linkString = this->matchLink(string);
 
+                if (string[0] == '@' && app->settings->usernameBold) {
+                    this->emplace<TextElement>(string, MessageElement::Text, textColor,
+                                               FontStyle::ChatMediumBold);
+                    string = "";
+                }
```

Comment:
Don't duplicate code for no reason. Instead add a FontStyle variable and use it in the emplace<TextElement> below.

`string = "";` is a terrible hack here!